packages/dev/src/quickstarts-data/mocks/yamls/example-learning-path-1.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: example-learning-path-part-1
+  annotations:
```

Comment:
we can remove these additional fields
apiVersion, kind, metadata.annotations, spec.type (unless you want to customize this), spec.version